src/index.js

```diff
@@ -0,0 +1,172 @@
+
+// proxy request. UI fuctionality event listeners section
+const getLocation = () => {
+    let location = document.getElementById("cityNameInput").value
+    const url ='http://127.0.0.1:5000/location';
+
+    return axios.get(url, {params:{q:location}})
+    .then((response)=>{
+
+    let latitude = response.data[0].lat;
+    let longitude = response.data[0].lon
+    console.log(longitude)
+    console.log(latitude)
+    return {latitude:latitude, longitude: longitude};
+
+
+    })
+    .catch( (error) => {
+        console.log('error in findLatitudeAndLongitude!');
+      });
+    
+ }
+
+const getWeather = (lat, lon) =>{
+    return axios
+    .get('http://127.0.0.1:5000/weather', {params:{lat:lat,lon:lon}})
+    .then((response)=>{
+       
+        let result = response.data['main']['temp'];
+        return result
+       
+    })
+    .catch((error)=>{
+        console.log(error);
+    }
+
+    )
```

Comment:
Nitpick - the parentheses should directly follow the right curly brace on line 35.